archive.sh

```diff
@@ -11,6 +11,27 @@ VERSION_RAW="$(sed -n 's|<version>\(.*\)</version>|\1|p' ./FinSearchUnified/plug
 VERSION="$(echo -e "${VERSION_RAW}" | tr -d '[:space:]')"
 echo "Version: ${VERSION}"
 
+echo $(git stash)
```

Comment:
I found a bug while click testing where this "causes an error" when there were no changes to be stashed. I would propose following approach:
1. Run `git stash` and save the return value.
2. Print the return value which should actually be a message.
3. Run `git stash pop` only if the previously saved return value equals `No local changes to save`.

Reply:
@ihanli I hope you mean when the return value does **NOT** equal to the the provided message else we don't need to apply the stash

Reply:
@zep-com 
Yes, that's what I meant.